enums.h

```diff
@@ -3102,7 +3102,12 @@ typedef enum _ui_classes {
 	UI_WARRIOR     = 0x0,
 	UI_ROGUE       = 0x1,
 	UI_SORCERER    = 0x2,
-	UI_NUM_CLASSES = 0x3,
+#ifdef HELLFIRE
+	UI_MONK             = 0x3,

```

Comment:
```suggestion
	UI_MONK        = 0x3,
```
Why would GitHub not use monospace in there editor :(

Reply:
:p

Reply:
going to see enums in my nightmares :p